### PR TITLE
chore: Bump Preact version

### PR DESCRIPTION
**Pull request by rschristian**
Devtools is failing on the [ecosystem-ci](https://github.com/preactjs/ecosystem-ci/actions/runs/11166359261) I had set up.

Two issues:
 - [preactjs/preact#4173](https://github.com/preactjs/preact/pull/4173) altered the structure of `preact/src/internal` to wrap all of the interfaces in `declare global`, so we can no longer access them directly
   - I see you've vendored some of the types for v11 here, but I went with `patch-package` for the moment. Happy to go with whatever you prefer, I absolutely can copy the types here if you'd like, import and use those types as globals, etc., just thought I'd throw out this approach first as it's the simplest and probably most straightforward.
 - [preactjs/preact#3690](https://github.com/preactjs/preact/pull/3690) improved the types for bare `h()` calls, so a couple of the tests where you add arbitrary props (`foo`, `__source`) now fail type checks.

### CODE REVIEW

**Comment by rschristian** [comment]
Didn't realize `yarn test`  didn't cover all tests. Looks like tests will need more extensive changes, will look into that more.

File: src/view/components/devtools.css [hunk 1]
```diff
@@ -165,7 +165,7 @@
 	z-index: 2;
 }
 
-.icon-btn:not([disabled])[data-active] .icon-btn-inner,
+.icon-btn:not([disabled])[data-active="true"] .icon-btn-inner,
```
Comment (rschristian) [inline src/view/components/devtools.css:168]: Helpful ripgrep pattern: `$ rg 'data-[\w-]+]'`

**Comment by rschristian** [comment]
Should be good to go. It was indeed the breaking change in data attributes & the addition of `.contains`

**Review by JoviDeCroock — APPROVED** [review]

**Review by marvinhagemeister — APPROVED** [review]

**Comment by rschristian** [comment]
@marvinhagemeister Just FYI, I'm not on the authorized users list here so I cannot merge.

Absolutely 0 rush to land this, I've switch the CI over to use this branch anyhow so it'll catch any future changes in Preact that may affect this repo, just wanted to mention it in case the intention was that I could merge whenever I thought it was good.

**Pull request merged**